### PR TITLE
[Console] Set Command::setHidden() final for adding default param in SF 6.0

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Would be nice, but that's a BC break... :)

**Comment by nicolas-grekas** [comment]
Could be made `@final since Symfony 5.1` to unlock the change for 6.0 (requires proper notes in the changelog+upgrade files of course)

**Comment by lyrixx** [comment]
OK, Fixed

**Review by nicolas-grekas — APPROVED** [review]

File: UPGRADE-6.0.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 UPGRADE FROM 5.x to 6.0
 =======================
 
+Console
+-------
+
+ * `Command::setHidden()` has a default value (`true`) for `$hidden` parameter
```
Comment (fabpot) [inline UPGRADE-6.0.md]: This should be somewhere in the code. If not, we will forget to add the default value in 6.0.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: UPGRADE-6.0.md (hunk 1 above)
Comment (lyrixx) [reply UPGRADE-6.0.md]: I added a note in the PHP Doc + `/*= true*/` in the method signature.

**Review by chalasr — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Pull request merged**